.travis.yml

```diff
@@ -59,22 +59,6 @@ jobs:
     - make e2e-test
   # split builds to avoid job timeouts
   - stage: publish amd64
-    if: type = api AND branch = master AND repo = kubernetes/ingress-nginx
+    if: type = push AND branch = master AND repo = Shopify/ingress
```

Comment:
can you temporarily remove conditioning on branch and see if it is actually pushing to our registry?